PoESkillTree.Engine.GameModel.Tests/PassiveTree/PassiveTreeDeserializationTest.cs

```diff
@@ -0,0 +1,63 @@
+﻿using FluentAssertions;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using PoESkillTree.Engine.GameModel.PassiveTree.Base;
+
+namespace PoESkillTree.Engine.GameModel.PassiveTree
+{
+    [TestFixture]
+    public class PassiveTreeDeserializationTest
+    {
+        [TestCase("skilltree_3.10.0.new.min.json")]
+        [TestCase("skilltree_3.10.0.old.min.json")]
+        [TestCase("skilltree_3.8.0.min.json")]
+        public void JsonPassiveTree_Deserialization_Serialization_Deserialization(string fileName)
```

Comment:
These three tests take around one second each (at least on [AppVeyor](https://ci.appveyor.com/project/PoESkillTree/poeskilltree-engine/builds/31474221/tests) and from what they do they're also more integration than unit tests.

It makes sense to create a PoESkillTree.Engine.GameModel.IntegrationTests project for them, imo, even if it's just for three test cases.

Reply:
Yep, they take a while, and I can move them out into an integration test. It is a nice to have. Just make sure we can deserialize our serealization as well as the other formats.

Reply:
A new project has been created and the 3 test cases moved there